panel/1.0/additional_configuration.md

```diff
@@ -127,7 +127,13 @@ reCAPTCHA can easily be disabled using the admin panel. In the Settings, select
 If you cannot access your panel, you can modify the database directly using the following commands.
 
 ```sql
+# If using MariaDB (V11.0.0+)
```

Comment:
Please use a lowercase `v` when referring to versions, otherwise my brain will explode :)

Also adding a note stating this is the default when following the docs may be useful so people don't get confused.

Reply:
Resolved in my latest commits